internal/envconfig/xds.go

```diff
@@ -53,4 +53,10 @@ var (
 
 	// C2PResolverTestOnlyTrafficDirectorURI is the TD URI for testing.
 	C2PResolverTestOnlyTrafficDirectorURI = os.Getenv("GRPC_TEST_ONLY_GOOGLE_C2P_RESOLVER_TRAFFIC_DIRECTOR_URI")
+
+	// XDSDualstackEndpointsEnabled is true if gRPC should read the
+	// "additional addresses" in the xDS endpoint resource.
+	// TODO: Control this using an env variable when all LB policies
+	// handle endpoints.
```

Comment:
I was discussing with @dfawley that only the primary address is present in the `resolver.Addresses` to ensure LB policies that don't read resolver.Endpoints continue to function correctly. E.g: Round robin will not create multiple subchannels with the same backend. He mentioned that if the env variable is exposed, users may assume that all LB policies handle endpoints correctly. So we decided to avoid adding release notes and exposing an env variable until the LB polices are updated.

I've opened an issue and added it to the TODO message.